resources.go

```diff
@@ -1293,3 +1244,192 @@ func FromName(options AutoNameOptions) func(res *tfbridge.PulumiResource) (inter
+		azureLegacyManagementGroups, azureManagement, nil)
+
+	// Migrate azureLegacyMgmtResource -> azureManagement
+	renameResourceWithAlias("azurerm_management_lock", "ManangementLock", "Lock",
+		azureLegacyMgmtResource, azureManagement, nil)
+
+	// Migrate `azurerm_event_grid_*` to new EventGrid Mod
+	renameResourceWithAlias("azurerm_eventgrid_domain", "Domain", "", azureEventHub, azureEventGrid,
+		nil)
+	renameResourceWithAlias("azurerm_eventgrid_event_subscription", "EventSubscription", "",
+		azureEventHub, azureEventGrid, nil)
+	renameResourceWithAlias("azurerm_eventgrid_topic", "EventGridTopic", "Topic", azureEventHub,
+		azureEventGrid, nil)
+
+	// Migrate `azurerm_servicebus_*` to new ServiceBus Mod
+	renameResourceWithAlias("azurerm_servicebus_namespace", "Namespace", "", azureEventHub,
+		azureServiceBus, &tfbridge.ResourceInfo{
+			Fields: map[string]*tfbridge.SchemaInfo{
+				// https://docs.microsoft.com/en-us/rest/api/servicebus/create-namespace
+				// Max length of a servicehub namespace is 50.
+				azureName: AutoNameWithMaxLength(azureName, 50),
+			},
+		},
+	)
+	renameResourceWithAlias("azurerm_servicebus_namespace_authorization_rule", "NamespaceAuthorizationRule",
+		"", azureEventHub, azureServiceBus, nil)
+	renameResourceWithAlias("azurerm_servicebus_queue", "Queue", "", azureEventHub, azureServiceBus,
+		&tfbridge.ResourceInfo{
+			Fields: map[string]*tfbridge.SchemaInfo{
+				// https://groups.google.com/forum/#!topic/particularsoftware/XuHp_8wZ09o
+				// Max length of a servicehub queue is 260.
+				azureName: AutoNameWithMaxLength(azureName, 260),
+			},
+		},
+	)
+	renameResourceWithAlias("azurerm_servicebus_queue_authorization_rule", "QueueAuthorizationRule",
+		"", azureEventHub, azureServiceBus, nil)
+	renameResourceWithAlias("azurerm_servicebus_subscription", "Subscription", "",
+		azureEventHub, azureServiceBus, nil)
+	renameResourceWithAlias("azurerm_servicebus_subscription_rule", "SubscriptionRule", "",
+		azureEventHub, azureServiceBus, nil)
+	renameResourceWithAlias("azurerm_servicebus_topic", "Topic", "", azureEventHub, azureServiceBus,
+		&tfbridge.ResourceInfo{
+			Fields: map[string]*tfbridge.SchemaInfo{
+				// https://groups.google.com/forum/#!topic/particularsoftware/XuHp_8wZ09o
+				// Max length of a servicehub topic is 260.
+				azureName: AutoNameWithMaxLength(azureName, 260),
+			},
+		},
+	)
+	renameResourceWithAlias("azurerm_servicebus_topic_authorization_rule", "TopicAuthorizationRule", "",
+		azureEventHub, azureServiceBus, nil)
+	renameDataSourceWithAlias("azurerm_servicebus_namespace", "getServiceBusNamespace", "getNamespace",
+		azureEventHub, azureServiceBus, nil)
+
+	// Migrate `azurerm_traffic_manager_*` to network module
+	renameResourceWithAlias("azurerm_traffic_manager_endpoint", "Endpoint", "TrafficManagerEndpoint",
+		azureLegacyTrafficManager, azureNetwork, nil)
+	renameResourceWithAlias("azurerm_traffic_manager_profile", "Profile", "TrafficManagerProfile",
```

Comment:
TF seems to have `TrafficManager` in its own module. Why do we move to `network`?

Reply:
Traffic manager is bundled as part of network in the Terraform docs - this is why we went with it

@lukehoban thoughts on this one?

Reply:
> TF seems to have TrafficManager in its own module. 

What do you mean by this?  TF has no concept of modules - right?

Reply:
Yup, given that it's under here:

![image](https://user-images.githubusercontent.com/4564579/62727675-d20d4f80-b9ce-11e9-905f-c5b3ef1e96d4.png)

I'm ok with this being in the 'network' module for us.

Reply:
Yeah, I guess we can ignore that. I just wasn't sure Traffic Manager belongs to network, but it might.